### DISCUSSION TITLE
Sprint 1 Retrospective

### DISCUSSION THREAD

**Discussion by annalieseschropp** [annalieseschropp/checkers#55] [Sprint Retrospective]
Hi everyone! I just wanted to open up a discussion to reflect on this first sprint now that we've submitted everything. So please answer the following questions so that we can improve things in the next sprint!

1. What went well in this sprint?
2. What could be improved upon in the next sprint?
3. What have you learned?
4. Anything else you want to say.

**Comment by annalieseschropp** [comment]
1. What went well:
    * I think we worked well as a team and split the work up quite evenly
    * I think our current project is something that is well organized and set up well for future features
    * I think we are using the tools within github well (i.e. the wiki, the issue tracking, the sprint board etc.)

2. What could be improved:
    * I think we need to meet more regularly to keep each other more informed on where we are within our tasks (QAD maybe?)
    * I think we should just make sure to branch off of dev from now on to prevent future rebasing 

3. What have I learned:
    * I learned how to use github for project management including sprint boards, wiki pages and tasks
    * I learned how to use Unity to set up prefabs, scripts and make something tangible in the project
    * I learned how to set up and create unity tests 

4. Anything else:
    * I just want everyone to know that I appreciate all their hard work on this and I'm excited for when the game is fully functional

**Comment by DJ-CO** [comment]
1. What went well in this sprint?
    * I feel that Unity collaboration went suspiciously well; I expected more unpleasant hitches.

2. What could be improved upon in the next sprint?
    * I’m hoping to see us tighten up our branching conventions, such as branching from dev, and naming branches using a “#.#_name” pattern.
    * I also would like to have more regular scrum meetings, and maybe even try to be as scrum-adherent as possible, just to see if it really makes a difference.

3. What have you learned?
    * I never knew that Github had such an expansive array of useful features.
    * Every game engine has its pros and cons.

4. Anything else you want to say.
    * Nothing else to say.

**Comment by imckechn** [comment]
1. What went well this sprint?
I think we kept to schedule really well. There was nobody that fell behind as far as I can tell and that really helped keep stress levels low since we were never feeling too rushed.

2. What could be improved upon in the next sprint.
I think our Git Branching could be a bit better. Using more clear names, detailing what they are for, and requesting pulls more frequently so everybody knows they are done and so they get reviewed.

3.  What have you learned?
I became very familiar with C# and using the Unity Editor. I am much more comfortable with the language and editor now.

4. Anything else you wanted to say?
Nothing else for me.
